Services/Administration/OJS.Services.Administration.Business/Validation/Factories/IProblemResourceValidatorsFactory.cs

```diff
@@ -0,0 +1,7 @@
+namespace OJS.Services.Administration.Business.Validation.Factories;
+
+using OJS.Data.Models.Problems;
+
+public interface IProblemResourceValidatorsFactory : IValidatorsFactory<ProblemResource>
```

Comment:
I hate these empty namespaces, and hated then since Interactive.


Can we try the following:
- `ProblemResourceValidatorsFactory: IValidatorsFactory<ProblemResource>`
- inject `IValidatorsFactory<ProblemResource>` 

Please, try

Reply:
I made separate issue: https://github.com/SoftUni-Internal/exam-systems-issues/issues/120